src/lxc/tools/lxc_unshare.c

```diff
@@ -44,6 +45,10 @@
 #include "network.h"
 #include "utils.h"
 
+
+static int wait_fd = -1;
+static uint64_t val = 1;
```

Comment:
Please don't make them global variables. Open the eventfd in main() and pass it as an argument to lxc_clone() by giving the args struct that is passed to main an additional argument. Same thing goes for the sync value.